apps/designer/app/designer/shared/tree/index.tsx

```diff
@@ -55,8 +57,22 @@ const instanceRelatedProps = {
 } as const;
 
 export const InstanceTree = (
-  props: Omit<TreeProps<Instance>, keyof typeof instanceRelatedProps>
-) => <Tree {...props} {...instanceRelatedProps} />;
+  props: Omit<
+    TreeProps<Instance>,
+    keyof typeof instanceRelatedProps | "findItemById"
+  >
+) => {
+  const { instancesById } = useStore(instancesIndexStore);
```

Comment:
Not a big deal, but this component is a bit weird now because it takes `rootInstance` as a prop, but takes `instancesById` directly from store.

I'd cleaned this up: either both from props or both from store